services/web/server/src/simcore_service_webserver/studies_access.py

```diff
@@ -0,0 +1,193 @@
+""" handles access to studies
+
+    Handles a request to share a given sharable study via '/study/{id}'
+
+    - defines '/study/{id}' routes (this route does NOT belong to restAPI)
+    - access to projects management subsystem
+    - access to statics
+    - access to security
+    - access to login
+
+FIXME: reduce modules coupling!
+TODO: THIS IS A PROTOTYPE!!!
+
+"""
+import json
+import logging
+import uuid
+from typing import Dict
+
+from aiohttp import web
+
+from .resources import resources
+from .security import is_anonymous, remember
+from .statics import index as app_index
+
+log = logging.getLogger(__name__)
+
+BASE_UUID = uuid.UUID("71e0eb5e-0797-4469-89ba-00a0df4d338a")
+
+
+def load_isan_template_uuids():
+    with resources.stream('data/fake-template-projects.isan.json') as fp:
+        data = json.load(fp)
+    return [prj['uuid'] for prj in data]
+
+ALLOWED_TEMPLATE_IDS = load_isan_template_uuids()
+
+
+# TODO: from .projects import get_template_project
```

Comment:
I need some more explanations about this file from you I think... I don't totally get the idea behind..

Reply:
This is still prototype ... I remind myself that I need to reduce couplgin and somehow this function should go in another subsystem